### PR TITLE
Improvements to trigger_multiple target entity conversion

### PR DESCRIPTION
**Pull request by Mac-tf2**

### CODE REVIEW

File: BSPConvertLib/Source/EntityConverter.cs [hunk 1]
```diff
@@ -404,11 +404,45 @@ private void ConvertTriggerMultiple(Entity trigger)
...
-
 			trigger["spawnflags"] = "1";
 		}
 
+		private void ConvertEntityTargets(Entity trigger, Entity entity)
+		{
+			var targets = GetTargetEntities(entity);
+			foreach (var target in targets)
+			{
+				switch (target.ClassName)
```
[the remark this answers is not in the document]
Comment (Panzerhandschuh) [reply BSPConvertLib/Source/EntityConverter.cs:417]: Can't you just call `ConvertEntityTargets(trigger, trigger)` at the start of `ConvertTriggerMultiple`?

File: BSPConvertLib/Source/EntityConverter.cs [hunk 2]
```diff
@@ -404,11 +404,45 @@ private void ConvertTriggerMultiple(Entity trigger)
 						OpenDoorOnStartTouch(trigger, target);
 						break;
 				}
+				ConvertEntityTargets(trigger, target);
 			}
-
 			trigger["spawnflags"] = "1";
 		}
 
+		private void ConvertEntityTargets(Entity trigger, Entity entity)
```
Comment (Panzerhandschuh) [inline BSPConvertLib/Source/EntityConverter.cs:412]: `ConvertTriggerTargetsRecursive` might be a better name

File: BSPConvertLib/Source/EntityConverter.cs [hunk 3]
```diff
@@ -566,17 +572,6 @@ private void GiveWeaponOnStartTouch(Entity trigger, Entity weaponEnt)
 			trigger.connections.Add(connection);
 
 			GiveWeaponAmmoOnStartTouch(trigger, weaponEnt);
-
-			var targets = GetTargetEntities(weaponEnt); //TODO: more robust solution for entities targeting other entities inside a trigger_multiple
```
Comment (Panzerhandschuh) [inline BSPConvertLib/Source/EntityConverter.cs:570]: Shouldn't you remove the `GetTargetEntities` code in `ConvertInitTrigger` as well?

Comment (Mac-tf2) [reply BSPConvertLib/Source/EntityConverter.cs:570]: forgot that was there, yep.

**Pull request merged**